lib/saxerator/latches/child_of.rb

```diff
@@ -31,7 +31,7 @@ def increment_depth(amount)
       end
 
       def depth_within_element
-        @depths.size > 0 ? @depths.last : 0
+        @depths.length > 0 ? @depths[-1] : 0
```

Comment:
That's not how aliases work - it's still a single method dispatch. Even that fast-ruby link says it was 1.01x slower, which I don't consider significant.